py-polars/Makefile

```diff
@@ -40,6 +40,7 @@ endif
 	$(VENV_ACT_BIN)pip install -U pip
 	$(VENV_ACT_BIN)pip install -r requirements-dev.txt
 	$(VENV_ACT_BIN)pip install -r requirements-lint.txt
+	$(VENV_ACT_BIN)pip install -r docs/requirements-docs.txt
```

Comment:
I left this out on purpose, as I think many contributors won't be building the docs locally. But I guess it doesn't really hurt. I should really work on the contributing guide.

Reply:
My thought (based on past experience) was _"never put any additional barriers in the way of contributing to the docs"_, heh.

Can remove if you prefer though; it's a one-off install after all, and I agree that the majority of contributions are going to come in the form of better docstrings rather than anything more structural ;)